client/src/pages/SignUpForm.jsx

```diff
@@ -79,161 +81,165 @@ const SignUpForm = () => {
-							name="email"
-							value={signUpData.email}
-							onChange={handleChange}
-							disabled
-							fullWidth
-						/>
-					</Box>
-					<Box mt={2}>
-						<TextField
-							label="Business Name"
-							variant="outlined"
-							name="businessName"
-							value={signUpData.businessName}
-							onChange={handleChange}
-							fullWidth
-						/>
-					</Box>
-					<Box mt={2}>
-						<TextField
-							label="Language *"
-							variant="outlined"
-							name="language"
-							value={signUpData.language}
-							onChange={handleChange}
-							fullWidth
-						/>
-					</Box>
-					<Box mt={2}>
-						<TextField
-							label="Profile Image"
-							variant="outlined"
-							name="profileImage"
-							value={signUpData.profileImage}
-							onChange={handleChange}
-							fullWidth
-						/>
-					</Box>
-					<Box mt={2}>
-						<TextField
-							label="Phone Number *"
-							variant="outlined"
-							name="phoneNumber"
-							value={signUpData.phoneNumber}
-							onChange={handleChange}
-							fullWidth
-						/>
-					</Box>
-					<Box mt={2}>
-						<TextField
-							label="Address *"
-							variant="outlined"
-							name="address"
-							value={signUpData.address}
-							onChange={handleChange}
-							fullWidth
-						/>
-					</Box>
-					<Box mt={2}>
-						<TextField
-							label="City *"
-							variant="outlined"
-							name="city"
-							value={signUpData.city}
-							onChange={handleChange}
-							fullWidth
-						/>
-					</Box>
-					<Box mt={2}>
-						<TextField
-							label="Country *"
-							variant="outlined"
-							name="country"
-							value={signUpData.country}
-							onChange={handleChange}
-							fullWidth
-						/>
-					</Box>
-					<Box mt={2}>
-						<TextField
-							label="Profession *"
-							variant="outlined"
-							name="profession"
-							value={signUpData.profession}
-							onChange={handleChange}
-							fullWidth
-						/>
-					</Box>
-					<Box mt={2}>
-						<TextField
-							label="Years of Experience *"
-							variant="outlined"
-							name="yearsOfExperience"
-							value={signUpData.yearsOfExperience}
-							onChange={handleChange}
-							fullWidth
-						/>
-					</Box>
-					<Box mt={2}>
-						<Typography variant="p" gutterBottom>
-							Hourly rate
-						</Typography>
-						<TextField
-							variant="outlined"
-							name="hourlyRate"
-							value={signUpData.hourlyRate}
-							fullWidth
-						/>
-					</Box>
-					<Box mt={2}>
-						<Button
-							type="submit"
-							variant="contained"
-							color="primary"
-							style={{ backgroundColor: "#F3263B" }}
-						>
-							Submit
-						</Button>
-					</Box>
-				</fieldset>
-			</form>
+						<Box mt={2}>
+							<Typography variant="p" gutterBottom>
+								First Name
+							</Typography>
+							<TextField
+								variant="outlined"
+								name="firstName"
+								value={signUpData.firstName}
+								onChange={handleChange}
+								fullWidth
+							/>
+						</Box>
```

Comment:
Why don't we have a function to build the element that repeats a lot

```javascript
function buildField(label, name, value, handler) {
    return <Box mt={2}>
							<Typography variant="p" gutterBottom>
								{label}
							</Typography>
							<TextField
								variant="outlined"
								name={name}
								value={value
								onChange={handleChange}
								fullWidth
							/>
  </Box>
}
//////////////////////
// then we can DRY our code up

{buildField('First Name', 'firstName', signUpData.firstName, handleChange)}
{buildField('Surname', 'lastName', signUpData.lastName, handleChange)}


```

Reply:
@karleenmsrichards and @Tony-devops would you like refactor your codes, base on this review. 
I just looked at functionality.

Reply:
@jarrodruby I can have that fixed today.

Reply:
so instead of making a stylized sub component this is the concept of a factory method